src/main/java/chanceCubes/rewards/defaultRewards/ItemRenamer.java

```diff
@@ -49,8 +50,8 @@ public void trigger(ServerLevel world, BlockPos pos, Player player, JsonObject s
 		{
 			ItemStack dirt = new ItemStack(Blocks.DIRT);
 			MutableComponent name = ComponentWrapper.string("A lonely piece of dirt");
-			name.setStyle(name.getStyle().withColor(TextColor.parseColor("#ff1111")));
-			dirt.setHoverName(name);
+			name.setStyle(name.getStyle().withColor(TextColor.parseColor("#ff1111").getOrThrow()));
```

Comment:
Technically fine, but probably best to not throw and instead just fall back to white or something

Reply:
Replaced it with what parseColor does internally. that way we don't need to deal with DataResult